### ISSUE TITLE
[Javascript]Action.ShowCard in Popup mode is not working

### ISSUE THREAD

**Issue by atishayv** [microsoft/AdaptiveCards#5450]
# Platform

- [ ] JavaScript

# Author or host

Host Name : Microsoft Search

# Version of SDK

"adaptivecards": "3.1.0-ci.202010.14001.0"

# Details

On changing the actionMode of showcard from inline to Popup in hostConfig,

```
actions: {
   ...
    showCard: {
      actionMode: "Popup",
    }
},
```

the ideal behaviour on click of action.showcard should be that `onExecuteAction` callback should be called and the card should not be shown so that the client can show the card in whichever way they want.

But the issue is that although the callback is getting called, the framework still shows the adaptive card inline.

So basically, it calls this : 

![image](https://user-images.githubusercontent.com/13940020/109596656-afa4cf00-7b3c-11eb-9c2f-62b9e6f51761.png)

and in the actionExecuted method, it renders the card inline.

Can this be fixed ?

**Comment by atishayv** [comment]
CC : @dclaux

**Comment by paulcam206** [comment]
@dclaux - care to take a look?

**Comment by jonmill** [comment]
@dclaux - are you able to investigate this one to see if it is a bug or should someone else take a look?

**Comment by dclaux** [comment]
Sorry for the delay on this. I'll fix it.
